cmake/cpu_features.cmake

```diff
@@ -30,7 +30,7 @@ if (ARCH_NATIVE)
     set (COMPILER_FLAGS "${COMPILER_FLAGS} -march=native")
 
 elseif (ARCH_AARCH64)
-    set (COMPILER_FLAGS "${COMPILER_FLAGS} -march=armv8-a+crc+simd+crypto+dotprod+ssbs")
+    set (COMPILER_FLAGS "${COMPILER_FLAGS} -march=armv8-a+crc+simd+crypto+dotprod+ssbs+lse")
```

Comment:
As I don't have permissions to push to your fork, would you like to add the following lines below l. 33?

```cmake
# The LSE extension for atomic operations is mandatory in ARM >= v8.1. As we target v8.0,
# use a runtime-dispatch that falls back to non-atomic operations hardware w/o LSE.
set (COMPILER_FLAGS "${COMPILER_FLAGS} -moutline-atomics")
```

Reply:
The -moutline-atomics was supported by clang-12(see [here](https://reviews.llvm.org/rG4d7df43ffdb460dddb2877a886f75f45c3fee188)) and gcc 10.1. The minimal compiler requirement of ClickHouse(gcc 11 or clang 12) contains this support. 
Compared to +lse, -moutline-atomics provides better compatibility with a little bit performance impact.